src/julia/darray.jl

```diff
@@ -128,6 +161,21 @@ function foxLi{T<:BlasComplex}(::Type{T}, n::Integer, ω::Real)
 end
 foxLi(n::Integer, ω::Real) = foxLi(Complex128, n, ω)
 
+function uniform{T<:BlasFloat}(::Type{T}, m::Integer, n::Integer)
```

Comment:
What guarantees does `drand` make about the independence of each process's random number stream? In C this leads to catastrophic problems.